docker/scripts/singularity_converter.py

```diff
@@ -0,0 +1,50 @@
+"""
+This script is a modification of the
+``spython recipe Dockerfile &> Singularity.def`` command, implemented here:
+github.com/singularityhub/singularity-cli/blob/master/spython/client/recipe.py
+
+It converts the Docker recipy to Singularity, but suppressing any %files
```

Comment:
Nit: recipy -> recipe

Reply:
Silly me! thanks for the quick review, I am happy if this helps a bit.